network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -21,7 +21,7 @@ <h1>{{ block.value }}</h1>
     </div>
 
     {% if page.cta != None %}
-    <div class="cta col-md-6">
+    <div class="cta col-lg-4 offset-lg-1">
```

Comment:
This looks like it breaks multipage campaigns. https://foundation-mofostaging-pr-1555.herokuapp.com/campaigns/multi-page/

Reply:
Cool cool, I'll look into that.

Reply:
Pushed an updated, should fix it.